docs/getting-started/create-an-app.md

```diff
@@ -106,6 +106,10 @@ The install process may also fail if no Python installation is available. Python
 is commonly available in most systems already, but if it isn't you can head for
 example [here](https://www.python.org/downloads/) to install it.
 
+#### Could not execute command yarn install
+
+Install yarn on your system with `sudo npm install --global yarn`
```

Comment:
It is not generally recommended to install any `npm` packages as `root` as that poses a security risk.

Can we avoid the `sudo`? If developers are facing issues with `EACCES` (which `sudo` solves) they should make sure their `npm` setup is correctly configured instead? See [Resolving EACCES permissions errors when installing packages globally](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally) in the npm Docs for more details.

Reply:
Perhaps we could link to https://classic.yarnpkg.com/en/docs/install/ here instead?

Reply:
sudo was removed, prerequisties page was referenced